tests/annotations/test_text.py

```diff
@@ -30,3 +33,46 @@ def test_pdf_object(self):
         assert obj.Rect == [x1, y1, x2, y2]
         assert obj.AP.N.BBox == [x1, y1, x2, y2]
         assert obj.AP.N.Matrix == translate(-x1, -y1)
+
+    def test_unicode_content(self):
+        # Python2.7 chokes on the string validation because we're passing in a unicode string
+        # But this functionality shouldn't be being used currently
+        if sys.version_info.major >= 3:
```

Comment:
The validation on strings chokes if you pass a unicode string into python 2.  I don't think the library gets used this way currently, so was debating dropping support for python 2, changing the validation, or not testing this unused scenario.  This seemed easiest.